code/game/machinery/bots/cleanbot.dm

```diff
@@ -374,11 +381,18 @@ text("<A href='?src=\ref[src];operation=oddbutton'>[src.oddbutton ? "Yes" : "No"
 		. = ..()
 
 /obj/machinery/bot/cleanbot/roomba/Crossed(atom/A)
+	if(coolingdown)
+		return
 	if(isliving(A))
 		var/mob/living/L = A
-		if(prob(30))
+		if(prob(10))
```

Comment:
undocumented behavior change?

Reply:
I mentioned that essentially if you have a higher chance, the roombas become truly hellish. I lowered the probability and introduced a cooldown to prevent players spamming the attack.

Reply:
As long as you put it in the CL

Reply:
Sorted